src/components/shared/Slider/Slider.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { render, cleanup } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import { Slider } from './Slider';
+
+describe('Stars score component', () => {
+  afterEach(cleanup);
+
+  it('renders stars with score 0', () => {
```

Comment:
```suggestion
  it('renders starts with score 0', () => {
```

Reply:
I'm not so sure it should start with 0, it could start at any value. For plans, for example they do not start at 0.